src/components/post-forms.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from "react";
+import "../App.css";
+import PostForm from "./post.form";
+
+class PostForms extends Component {
```

Comment:
You have two components with VERY similar names - `PostForms` and `PostForm`. please reconsider your component names